simulator/src/main/java/com/clearspring/analytics/stream/frequency/CountMinTinyLFU.java

```diff
@@ -0,0 +1,56 @@
+package com.clearspring.analytics.stream.frequency;
+
+/**
+ * A version of the TinyLFU sketch based on a regular conservative update sketch. 
+ * The difference is that anytime the sum of counters reach a predefined values
+ * we divide all counters by 2 in what is called a reset operation. 
+ * 
+ * @author gilga
+ *  [1] TinyLFU: A Highly Efficient Cache Admission Policy
+ *  http://www.cs.technion.ac.il/~gilga/TinyLFU_PDP2014.pdf
+
+ */
+public class CountMinTinyLFU extends ConservativeAddSketch{
```

Comment:
`LFU` --> `Lfu` to conform to Java conventions for abbreviations in camel-case names.